ui/src/state/lure/lure.ts

```diff
@@ -64,6 +65,20 @@ export function useLureWelcome(name: string): [string, (s: string) => void] {
   return [lureWelcome, setLureWelcome];
 }
 
+export function useGroupInviteUrl(flag: string): [string, () => void] {
```

Comment:
We should make sure that you only ask for the invite url when you enter a specific group.  Currently, it asks for it for all groups at once.  We can verify the behavior with `:reel +verb` and `:grouper +verb`